wunderio/next4drupal-project
# Nex 9  - translation support

*Link to ticket: [NEX-9](https://wunder.atlassian.net/jira/software/c/projects/NEX/boards/389?modal=detail&selectedIssue=NEX-9)*

*Changes proposed in this PR:*
- Recipe wunder_languages added. It adds basic language setup for the site. 
- Custom module wunder_translations with the command added. It allows us to enable translations for Article and Basic page content types.
- README instructions are updated to set up the project
- Next (frontend): locale is added. Translated versions of articles are thus shown.

*How to test:*
1. Start from scratch and follow the instructions from an updated README file of the project.
2. Create Articles and translate into fi/sv
3. Open Next Preview / localhost:3000 and confirm you can see the translated versions of the articles.

*Link to feature environment:*
<add silta url here>
